packages/deployments/contracts/contracts/lib/Connext/AssetLogic.sol

```diff
@@ -31,7 +31,7 @@ library AssetLogic {
     uint256 _assetAmount,
     uint256 _fee,
     IWrapped _wrapper
-  ) external returns (address, uint256) {
+  ) internal returns (address, uint256) {
```

Comment:
I had eliminated AssetLogic (https://github.com/connext/nxtp/pull/1015/commits/0d7dd679017fbbf275f561cc006946bcb9a001e3) and moved is content to ConnextUtils to avoid doing double delegated calls, to be honest I like much more having this separated AssetLogic but we meed all its functions to be `internal` make this an embed library instead of an linked one